packages/sdk/src/Actions/EventAction.ts

```diff
@@ -588,18 +723,11 @@ export class EventAction extends DeployableTarget<
    * @public
    * @async
    * @param {ActionStep} actionStep - The action step containing the event to validate.
-   * @param {ValidateActionStepParams} params - Additional parameters for validation, including known events and logs
+   * @param {EventLogs} logs - Event logs to validate the given step against
    * @returns {Promise<boolean>} Resolves to true if the action event is valid, throws if input is invalid, otherwise false.
    */
-  public isActionEventValid(
-    actionStep: ActionStep,
-    params: ValidateActionStepParams,
-  ) {
-    if (!('logs' in params)) {
-      throw new ValidationLogsMissingError();
```

Comment:
We can remove this error since this was the only place it was used

Reply:
removed